hybrasyl/World.cs

```diff
@@ -4366,7 +4467,7 @@ public object ScriptMethod(string name, params object[] args)
 
         private void QueueConsumer()
         {
-            while (!MessageQueue.IsCompleted)
+            while (!MessageQueue.IsAddingCompleted)

```

Comment:
Why is this changed? `IsCompleted` implies `IsAddingCompleted` plus the queue being empty. In general I feel like we want both of those things, especially for certain types of messages (checkpointing, etc).

Reply:
This was changed during testing. Will revert.